python/tvm/_ffi/_ctypes/object.py

```diff
@@ -106,6 +106,11 @@ class ObjectBase(object):
 
     def __del__(self):
         if _LIB is not None:
+            try:
+                handle = self.handle
```

Comment:
We can use hasattr(self, “handle”)

Reply:
I thought about that but I think hasattr() is slow and this way we only enter the slow case when bad things happen

Reply:
gotcha. Thanks!